src/js/actions/layers.js

```diff
@@ -1553,7 +1553,7 @@ define(function (require, exports) {
             return layer.isBackground;
         });
         
-        options = _.merge({ coalesce: false }, options);
+        options = _.merge({}, options);
```

Comment:
At first glance this seems like it might change the behavior ... but I guess maybe you confirmed is that there is no reason to supply a "default value" of false *here* because that is the default option further down the stack?